examples/macro.rs

```diff
@@ -0,0 +1,57 @@
+#[macro_use(rspec_run)]
```

Comment:
please comment the objective of this example, what it does, why it's great

Reply:
maybe rename the example ? Macro is not clear what it does. With a good name, it may not need a top comment.

I think it a great showcase of the environment...